<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>515</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>515</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>515</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Web Spider</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="SearchTextLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Search Text&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>600</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ResultLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Results:</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>600</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>600</width>
      <height>261</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>611</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::SelectedClicked</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column>
     <property name="text">
      <string>URL</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="StartURLLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>46</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Start URL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="StartURL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>600</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>235</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="UrlLabel">
         <property name="text">
          <string>URL count</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="UrlSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2048</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="ThreadLabel">
         <property name="text">
          <string>Thread count</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="ThreadSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>2048</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>460</y>
      <width>239</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="StartButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PauseButton">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="resultURLCount">
         <property name="text">
          <string>Url checked:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resultURLCountValue">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="resultFoundCount">
         <property name="text">
          <string>Text Found:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="resultFoundCountValue">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>StartURL</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>UrlSpinBox</tabstop>
  <tabstop>ThreadSpinBox</tabstop>
  <tabstop>StartButton</tabstop>
  <tabstop>PauseButton</tabstop>
  <tabstop>CancelButton</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
